node-group-project/src/components/PetShow.js

```diff
@@ -13,16 +14,24 @@ const PetShow = (props) => {
     .then(result => result.json())
     .then(pet => {
       setPet(pet[0])
-      console.log(pet)
     })
   }, []);
   
   const handleAdoptClick = () => {
     let formState = displayForm === true ? false : true
     setDisplayForm(formState)
   }
+
+  let adoptForm;
+  if (applicationStatus === "pending") adoptForm = "Application status Pending"
+  else {
+    adoptForm = displayForm === true ? <
```

Comment:
If possible the < of the PetAdoptionsForm component tag should be attached to the rest of it on the next line.